src/sentry/static/sentry/app/views/projectDebugFiles.jsx

```diff
@@ -175,35 +202,42 @@ const ProjectDebugSymbols = createReactClass({
             {dsym.symbolType === 'proguard' && dsym.objectName === 'proguard-mapping'
               ? '-'
               : dsym.objectName}
-            <p className="m-b-0 text-light small">
+            <DebugSymbolDetails className="text-light small">
               {dsym.symbolType === 'proguard' && dsym.cpuName === 'any'
                 ? 'proguard mapping'
                 : `${dsym.cpuName} (${symbolType})`}
-            </p>
+
+              {features &&
+                features.map(feature => (
+                  <Tooltip key={feature} title={getFeatureTooltip(feature)}>
+                    <span>
```

Comment:
`Tooltip` internally assigns a `ref` to the direct child element. This requires a DOM element.